File: examples/python/asm-raw-script/Anyone.runar.py
# Anyone — minimal `asm` raw-script contract (Python surface).
from runar import UnsafeSmartContract, public, asm


class Anyone(UnsafeSmartContract):
    def __init__(self):
        super().__init__()

    @public
    def unlock(self):
        asm("51", 0, 1)

File: examples/zig/asm-raw-script/Anyone.runar.zig
// Anyone — minimal `asm` raw-script contract (Zig surface).
const runar = @import("runar");

pub const Anyone = struct {
    pub const Contract = runar.UnsafeSmartContract;

    pub fn init() Anyone {
        return .{};
    }

    pub fn unlock(self: *const Anyone) void {
        _ = self;
        runar.asm("51", 0, 1);
    }
};
